Bottom: 42b00c42573d8f82fb86d29b2c411844d42adbe8
Top:    2b6891dabade51a32e90e756e066e1341785d115
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-05-11 15:39:53 +0100

Refresh of fix-poly_contourincontour-test

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 9c357f1..c2371b5 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2256,8 +2256,9 @@ poly_M_CheckInside (POLYAREA * p, Vector v0)
 
 /* NB: This function assumes the caller _knows_ the contours do not
  *     intersect. If the contours intersect, the result is undefined.
- *     This function will return an incorrect results if the contours
- *     share a commnon node at the arbitrary point tested.
+ *     Tt will return the correct result if the contours which share
+ *     common points beteween their contours. (Two identical contours
+ *     are treated as being not inside each other).
  */
 int
 poly_ContourInContour (PLINE * poly, PLINE * inner)
